samples/correlation-csharp/FunctionAppCorrelation/EternalOrchestrations.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+namespace FunctionAppCorrelation
+{
+    public class EternalOrchestrations
+    {
+        private readonly TelemetryClient telemetryClient;
+
+        public EternalOrchestrations(TelemetryConfiguration telemetryConfiguration)
+        {
+            this.telemetryClient = new TelemetryClient(telemetryConfiguration);
+        }
+
+        [FunctionName(nameof(Periodic_Cleanup_Loop))]
+        public async Task Periodic_Cleanup_Loop(
+            [OrchestrationTrigger] IDurableOrchestrationContext context)
+        {
+            var expire = context.GetInput<Expire>();
+            await context.CallActivityAsync(nameof(this.CleanUpNotification), "CleanUp!");
+            DateTime nextCleanup = context.CurrentUtcDateTime.AddSeconds(10);
+            if (expire.Value > nextCleanup)
+            {
+                await context.CreateTimer(nextCleanup, CancellationToken.None);
+                context.ContinueAsNew(expire);
+            }
+        }
+
+        [FunctionName(nameof(CleanUpNotification))]
+        public void CleanUpNotification(
+            [ActivityTrigger] string message)
+        {
+            this.telemetryClient.TrackTrace(message);
```

Comment:
It would be nice to comment when you want to use the telemetry client explicitly like this, vs what is handled automatically.

Reply:
Added!

Reply:
The GitHub preview UI just hasn't updated though...